Formula/postgresql@9.6.rb

```diff
@@ -23,7 +24,7 @@ class PostgresqlAT96 < Formula
   keg_only :versioned_formula
 
   # https://www.postgresql.org/support/versioning/
-  deprecate! date: "2021-11-11", because: :unsupported
+  disable! date: "2021-11-11", because: :unsupported
```

Comment:
Worth noting that this will prevent users from using `brew postgresql-upgrade-database` from this version, and we haven't been warning them of the upcoming deprecation.

Reply:
Yeah it sucks, but I don't really want to support it longer either

Reply:
We shouldn't disable something with what will now be zero deprecation notice, though.